src/components/CodeHighlight/Languages/langJSON.js

```diff
@@ -0,0 +1,19 @@
+export const langJSON = (code, styles) => {
+  const codeString = typeof code === 'string' ? code : JSON.stringify(code, null, 2);
+  const json = codeString.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/>/g, '&gt;');
+  const re = /("(\\u[a-zA-Z0-9]{4}|\\[^u]|[^\\"])*"(\s*:)?|\b(true|false|null)\b|-?\d+(?:\.\d*)?(?:[eE][+-]?\d+)?)/g;
+
+  return json.replace(re, match => {
+    let cls = 'number';
```

Comment:
Please give a more understandable name. Does it mean `class`?

Reply:
Done